### ISSUE TITLE
mbpfan.service works great, but not starting on boot

### ISSUE THREAD

**Issue by mrjoecastellon** [linux-on-mac/mbpfan#255]
TL;DR
mbpfan runs spectacularly well on my 2015 base model MacBook Pro 13. The service ALMOST never starts at boot though.

I am running Arch Linux on Endeavour OS. I installed the AUR package through `yay -S mbpfan-git`

I followed the following steps to install the service for boot:
```
sudo cp /usr/lib/systemd/system/mbpfan.service /etc/systemd/system
sudo systemctl enable mbpfan.service
sudo systemctl daemon-reload
sudo systemctl start mbpfan.service
```

All steps completed with no errors or warnings.

I always check the service after a reboot, and so far, out of seemingly 30+ reboots, I have only seen the service start automatically 2 times, and at this point, I'm not even certain if those were actually reboots or just me signing out and signing back in, perhaps with the service already running in the background.

Immediately after a reboot, the service status shows as follows:
![image](https://user-images.githubusercontent.com/91094235/197315658-6c83f306-9f12-42bf-a986-b6bdca87b9e6.png)

I then simply run `sudo systemctl start mbpfan.service` and the service starts normally.
![image](https://user-images.githubusercontent.com/91094235/197316039-ba1cc1b7-f1e5-4515-84d0-f19f699d6a23.png)

I hope someone can tell me what I'm doing wrong :)

**Comment by heimoshuiyu** [comment]
This problem was fixed in PR <https://github.com/linux-on-mac/mbpfan/pull/253> .

However the aur package is not updated to install the file `/lib/modules-load.d/mbpfan.depend.conf`.

You may manually copy this file to your system, or modify the `PKGBUILD` file in aur, or get rid of aur and install mbpfan using `make install`

**Comment by mrjoecastellon** [comment]
> This problem was fixed in PR #253 .
> 
> However the aur package is not updated to install the file `/lib/modules-load.d/mbpfan.depend.conf`.
> 
> You may manually copy this file to your system, or modify the `PKGBUILD` file in aur, or get rid of aur and install mbpfan using `make install`

Thank you very much!

**Comment by heimoshuiyu** [comment]
The aur package already fixed this issue <https://aur.archlinux.org/cgit/aur.git/commit/?h=mbpfan-git&id=03288aa0926e980fbd456665047a4bda5fc9c5fd>
